### PR TITLE
Add use case workflow

### PR DESCRIPTION
**Pull request by dianaboiangiu**

### CODE REVIEW

File: use_cases/models.py [hunk 1]
```diff
@@ -44,6 +44,24 @@ def __str__(self):
         return self.name
 
 
+def check_owner_user(instance, user):
+    return user == instance.created_by
+
+
+def check_supervisor_user(instance, user):
+    return user.is_superuser
```
Comment (gabrielastrezea) [inline use_cases/models.py:52]: are we sure we want to use the superuser here? I'm asking cause at the moment there are only 3 people that can do this action.

File: insitu/static/js/use-cases-detail.js
Comment (gabrielastrezea) [inline insitu/static/js/use-cases-detail.js:1]: not that important, but it seams like all of the object related js files have _ in their name 😄

**Review by gabrielastrezea — APPROVED** [review]
Looks great!

File: use_cases/models.py (hunk 1 above)
Comment (dianaboiangiu) [reply use_cases/models.py:52]: I'll add a group and we'll change it later if required.

**Pull request merged**